app/controllers/settings/password.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
+
+export default Controller.extend({
+  actions: {
+    changePassword(passwordData) {
+      let payload = {
+        'data': {
+          'old-password' : passwordData.passwordCurrent,
+          'new-password' : passwordData.passwordNew
+        }
+      };
+      this.get('loader')
+        .post('/auth/change-password', payload)
+        .then(data => {
+          if (data.error) {
+            this.get('notify').error(this.l10n.t(data.error));
+          } else {
+            this.get('notify').success(this.l10n.t('Password updated successfully'));
```

Comment:
@sumedh123 You can also add a loader to make sure the user does not make another request like other forms.